config.rb

```diff
@@ -108,4 +108,10 @@ def self.test?
   optional :postgres_service_blob_storage_endpoint, string
   optional :postgres_service_blob_storage_access_key, string
   optional :postgres_service_blob_storage_secret_key, string
+
+  # Logging
+  # :nocov:
+  override :database_logger, (test? ? :fatal : nil), logger
```

Comment:
Why do we need the test/fatal branch?

Reply:
This is its current behavior. We can disable it by default as well.
However, we need to delete the following line; otherwise, it will append the DEBUG logger.

https://github.com/ubicloud/ubicloud/blob/22cf0a075e7b556b67f3b39d5d7e214c25bcc1ff/spec/spec_helper.rb#L55